index.html

```diff
@@ -21,13 +21,31 @@ <h1>Michi Berger</h1>
     <img src="https://source.unsplash.com/random/600x300" alt="">
     <figcaption>A random picture from unsplash.com</figcaption>
   </figure>
-  
+
+  <!-- skills section - e.g. skills (dev tools, dev languages), hobbies or interests -->
+  <section id="skills"></section>
+
+  <!-- contact section - address, email -->
   <section id="contact">
     <h2>Contact</h2>
   </section>
+
+  <!-- recent-work section - github, codepen,  -->
+  <section id="recent-work"></section>
+
+  <!-- about-me section - short intro about me -->
+  <section id="about-me"></section>
+
+  <!-- biography section - working experience, education -->
   <section id="biography">
     <h2>Biography</h2>
   </section>
+
+  <!-- footer and social-media section e.g. linkedIn, instagram, facebook -->
+  <footer id="footer">
```

Comment:
not sure if the footer needs to have the id. But I think you have more knowledge, so it might be right :-)

Reply:
> 
Well written code in general and nice with the comments! I think you dont need the ID but you can use it to style the footer with css later on.

Reply:
thx Jelena for the review :) actually I´m not sure about the id :D I think it´s not wrong to place one...maybe with a better "semantic" name